tests-behat/bootstrap/Context.php

```diff
@@ -452,8 +452,7 @@ protected function getFinishedScript(): string
     {
         return 'document.readyState === \'complete\''
             . ' && typeof jQuery !== \'undefined\' && jQuery.active === 0'
-            . ' && typeof atk !== \'undefined\' && atk.vueService.areComponentsLoaded()'
-            . ' && jQuery(\':animated\').length === 0';
```

Comment:
everything is tweaked to not need it, so we want to keep that clean state if possible